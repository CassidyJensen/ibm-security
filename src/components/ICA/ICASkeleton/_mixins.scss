////
/// Important Content Area (ICA) skeleton mixins.
/// @group ICA
/// @copyright IBM Security 2019
////

@import '../../../globals/index';

@mixin ica-skeleton {
  width: carbon--mini-units(8);

  & &__label,
  & &__value {
    @include skeleton;
  }

  & &__label {
    height: carbon--mini-units(1.5);
    margin-top: 0;
    margin-bottom: carbon--mini-units(1);
  }

  & &__value {
    height: carbon--mini-units(4);
    margin: 0;
  }
}
