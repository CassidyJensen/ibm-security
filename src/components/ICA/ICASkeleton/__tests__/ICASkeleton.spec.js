/**
 * @file Important Content Area (ICA) skeleton tests.
 * @copyright IBM Security 2019
 */

import React from 'react';
import { shallow } from 'enzyme';
import ICASkeleton from '../ICASkeleton';

describe('ICASkeleton', () => {
  it('matches default snapshot', () =>
    expect(shallow(<ICASkeleton />)).toMatchSnapshot());
});
