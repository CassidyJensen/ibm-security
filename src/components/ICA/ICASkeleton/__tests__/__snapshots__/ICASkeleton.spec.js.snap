// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ICASkeleton matches default snapshot 1`] = `
<div
  className="security--ica-skeleton"
>
  <SkeletonText
    className="security--ica-skeleton__label"
    heading={false}
    lineCount={3}
    paragraph={false}
    width="100%"
  />
  <SkeletonText
    className="security--ica-skeleton__value"
    heading={true}
    lineCount={3}
    paragraph={false}
    width="80%"
  />
</div>
`;
